### PR TITLE
RSWW-9 added description and rating in tour scraping 

### PR DESCRIPTION
**Pull request by CzajaLukasz**

### CODE REVIEW

File: Itaka scraper/scrapping_single_tour.py [hunk 1]
```diff
@@ -44,6 +44,20 @@ def get_tour_name(driver: WebDriver) -> Optional[str]:
...
+def get_tour_rating(driver: WebDriver) -> Optional[str]:
+    rating_xpath = ("/html/body/div[5]/div[4]/div[2]/div/div[1]/div[3]/div[1]/div[2]/div[2]/span/span[2]/span[2]"
+                    "/span[2]/div[2]/div/span[2]/span[2]/div/div[1]/span/strong")
+    rating = scrape_element_by_xpath(driver, rating_xpath)
+    return rating
+
+
+def get_tour_description(driver: WebDriver) -> Optional[str]:
+    description_xpath = ("/html/body/div[5]/div[4]/div[2]/div/div[1]/div[3]/div[1]/div[5]/div/section/span/span[2]"
+                         "/div/div/p/text()")
```
Comment (pcichowski) [inline Itaka scraper/scrapping_single_tour.py]: `text()` seems suspicious for an Xpath, I think extracting text should / *might* be done using `WebElement.text`

**Review by pcichowski — COMMENTED** [review]
doesn't work on my end
```
Traceback (most recent call last):
  File "/home/silentsky/uni/rsww/scraper/logic/tours/scrapping_single_tour.py", line 329, in <module>
    test_tour = scrape_single_tour(driver, test_tour)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/silentsky/uni/rsww/scraper/logic/tours/scrapping_single_tour.py", line 310, in scrape_single_tour
    tour.description = get_tour_description(driver)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/silentsky/uni/rsww/scraper/logic/tours/scrapping_single_tour.py", line 57, in get_tour_description
    description = scrape_element_by_xpath(driver, description_xpath)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/silentsky/uni/rsww/scraper/logic/tours/scrapping_single_tour.py", line 22, in scrape_element_by_xpath
    element = driver.find_element(By.XPATH, element_xpath)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/silentsky/uni/rsww/scraper/.venv/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py", line 741, in find_element
    return self.execute(Command.FIND_ELEMENT, {"using": by, "value": value})["value"]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/silentsky/uni/rsww/scraper/.venv/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py", line 347, in execute
    self.error_handler.check_response(response)
  File "/home/silentsky/uni/rsww/scraper/.venv/lib/python3.11/site-packages/selenium/webdriver/remote/errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.JavascriptException: Message: Cyclic object value
```

File: logic/tours/scrapping_single_tour.py [hunk 2]
```diff
@@ -44,6 +44,20 @@ def get_tour_name(driver: WebDriver) -> Optional[str]:
...
+def get_tour_rating(driver: WebDriver) -> Optional[str]:
+    rating_xpath = ("/html/body/div[5]/div[4]/div[2]/div/div[1]/div[3]/div[1]/div[2]/div[2]/span/span[2]/span[2]"
+                    "/span[2]/div[2]/div/span[2]/span[2]/div/div[1]/span/strong")
+    rating = scrape_element_by_xpath(driver, rating_xpath)
+    return rating
+
+
+def get_tour_description(driver: WebDriver) -> Optional[str]:
+    description_xpath = ("/html/body/div[5]/div[4]/div[2]/div/div[1]/div[3]/div[1]/div[5]/div/section/span/span[2]"
+                         "/div/div/p/text()")
```
Comment (pcichowski) [inline logic/tours/scrapping_single_tour.py]: `text()` seems suspicious for an Xpath, I think extracting text should / *might* be done using `WebElement.text`

**Comment by CzajaLukasz** [comment]
Done

**Review by Mixss — APPROVED** [review]
działa

**Pull request merged**